### PR TITLE
mods_update(Enhanced Block Entities): 更新至 v0.9

### PR DESCRIPTION
**Pull request by xMikux**

### CODE REVIEW

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 1]
```diff
@@ -32,6 +34,8 @@
     "option.ebe.bed_ao.comment": "允許床使用平滑光源與環境遮蔽。需啟用「最佳化床」。",
     "option.ebe.shulker_box_ao": "平滑界伏盒光源",
     "option.ebe.shulker_box_ao.comment": "允許界伏盒使用平滑光源與環境遮蔽。需啟用「最佳化界伏盒」。",
+    "option.ebe.decorated_pot_ao": "平滑裝飾性盆栽光源",
```
Comment (notlin4) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:37]: "option.ebe.decorated_pot_ao": "平滑飾紋陶罐光源",

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 2]
```diff
@@ -32,6 +34,8 @@
...
+    "option.ebe.decorated_pot_ao.comment": "允許裝飾性盆栽使用平滑光源與環境遮蔽。需啟用「最佳化裝飾性盆栽」。",
```
Comment (notlin4) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:38]: Decorated Pot 的名稱為「飾紋陶罐」

**Review by notlin4 — CHANGES_REQUESTED** [review]

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 3]
```diff
@@ -13,15 +13,17 @@
...
     "option.ebe.render_enhanced_bells": "最佳化鐘",
-    "option.ebe.render_enhanced_bells.comment": "啟用完全處理過的鐘模型，在擁有大量鐘的區域大幅度提高效能。同時允許從遠處看到鐘。如果鐘變成隱形的，可能是由其他模組造成的問題，請停用。",
+    "option.ebe.render_enhanced_bells.comment": "啟用完全處理過的鐘模型，在擁有大量鐘的區域大幅度提高效能。同時允許從遠處看到鐘。如果鐘因為其他模組變成隱形，請停用此最佳化。",
     "option.ebe.render_enhanced_beds": "最佳化床",
-    "option.ebe.render_enhanced_beds.comment": "啟用已處理過的床模型，在擁有大量床的區域大幅度提高效能。同時允許從遠處看到床。如果床變成隱形的，可能是由其他模組造成的問題，請停用。",
+    "option.ebe.render_enhanced_beds.comment": "啟用已處理過的床模型，在擁有大量床的區域大幅度提高效能。同時允許從遠處看到床。如果床因為其他模組變成隱形，請停用此最佳化。",
     "option.ebe.render_enhanced_shulker_boxes": "最佳化界伏盒",
-    "option.ebe.render_enhanced_shulker_boxes.comment": "啟用已處理過的界伏盒模型，在擁有大量界伏盒的區域大幅度提高效能。同時允許從遠處看到界伏盒。如果界伏盒變成隱形的，可能是由其他模組造成的問題，請停用。",
+    "option.ebe.render_enhanced_shulker_boxes.comment": "啟用已處理過的界伏盒模型，在擁有大量界伏盒的區域大幅度提高效能。同時允許從遠處看到界伏盒。如果界伏盒因為其他模組變成隱形，請停用此最佳化。",
+    "option.ebe.render_enhanced_decorated_pots": "最佳化裝飾性盆栽",
```
Comment (StarskyXIII) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:25]:
```suggestion
    "option.ebe.render_enhanced_decorated_pots": "最佳化飾紋陶罐",
```

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 4]
```diff
@@ -13,15 +13,17 @@
...
+    "option.ebe.render_enhanced_decorated_pots.comment": "啟用已處理過的裝飾性盆栽模型，在擁有大量裝飾性盆栽的區域大幅度提高效能。同時允許從遠處看到裝飾性盆栽。如果裝飾性盆栽變成隱形，或是模組版的盆栽圖樣沒有正確運作，請停用此最佳化。",
```
Comment (StarskyXIII) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:26]:
```suggestion
    "option.ebe.render_enhanced_decorated_pots.comment": "啟用已處理過的飾紋陶罐模型，在擁有大量飾紋陶罐的區域大幅度提高效能。同時允許從遠處看到飾紋陶罐。如果飾紋陶罐變成隱形，或是其他模組的陶罐紋樣沒有正確運作，請停用此最佳化。",
```

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json (hunk 1 above)
Comment (StarskyXIII) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:37]:
```suggestion
    "option.ebe.decorated_pot_ao": "平滑飾紋陶罐光源",
```

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 5]
```diff
@@ -50,6 +54,8 @@
     "option.ebe.experimental_beds.comment": "啟用非常實驗性的繪製床功能，例如可與資源包配合使用的特別產生粒子紋理。",
     "option.ebe.experimental_signs": "實驗性告示牌",
     "option.ebe.experimental_signs.comment": "啟用非常實驗性的繪製告示牌功能，例如可與資源包配合使用的特別產生粒子紋理。",
+    "option.ebe.force_resource_pack_compat": "強制資源包相容性",
+    "option.ebe.force_resource_pack_compat.comment": "啟用這個當使用資源包時方塊變成隱形的情況。不要在資源包已設計為此模組情況下啟用。如果問題仍繼續發生，請回報給它們。",
```
Comment (StarskyXIII) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:58]:
```suggestion
    "option.ebe.force_resource_pack_compat.comment": "當使用的資源包造成方塊隱形時，請啟用該功能。不要在資源包已為此模組設計的情況下啟用。如果問題仍持續發生，請向資源包作者回報。",
```

File: MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json [hunk 6]
```diff
@@ -62,6 +68,7 @@
     "text.ebe.bell_options": "鐘選項",
     "text.ebe.bed_options": "床選項",
     "text.ebe.shulker_box_options": "界伏盒選項",
+    "text.ebe.decorated_pot_options": "裝飾性盆栽選項",
```
Comment (StarskyXIII) [inline MultiVersions/Fabric/main/enhancedblockentities/lang/zh_tw.json:71]:
```suggestion
    "text.ebe.decorated_pot_options": "飾紋陶罐選項",
```

**Review by StarskyXIII — CHANGES_REQUESTED** [review]

**Pull request merged**